test/unit/runner_tests.rb

```diff
@@ -35,7 +35,7 @@ class InitTests < UnitTests
       @runner = @runner_class.new(@options)
     end
     teardown do
-      subject.shutdown(0)
+      subject.shutdown(0) rescue false
```

Comment:
@kellyredding - This is because there is a test that makes sure if the `queue.shutdown` raises an error that it forces the workers to shutdown. After running that test, this will raise an exception. 

The goal of this is to end any threads that were spun up so they aren't taking resources while the test suite is running. The shutdown tests should let me know of any problems with shutting down a runner.